### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by dnlyun**
**Description**
- Fix some grammar

**Motivation and Context**
- Make doc look nice

### CODE REVIEW

File: docs/Changelog.md [hunk 1]
```diff
@@ -8926,6 +8926,8 @@ This version of the operator has been available since version 9 of the default O
 #### Type Constraints
 
 <dl>
+<dt><tt>I</tt> : tensor(int64)</dt>
+<dd>Int64 tensor</dd>
```
Comment (jcwchen) [inline docs/Changelog.md:8930]: Where did these updates come from? I didn't see corresponding update from defs.cc/old.cc

File: docs/TestCoverage.md [hunk 2]
```diff
@@ -15625,710 +15625,3 @@ expect(node, inputs=[x, y], outputs=[z],
...
-</details>
-<details>
-<summary>Unsqueeze: 1 out of 0 attributes covered</summary>
-
-</details>
-</details>
-
-
-# Overall Test Coverage
-## To be filled.
```
Comment (jcwchen) [inline docs/TestCoverage.md:16334]: The updates here in TestCoverage.md seem unrelated to this PR.

**Comment by jcwchen** [comment]
@dnlyun Do you have bandwidth to finish this PR? If not, please allow me to send a duplicate PR of yours and go forward. Thank you for your contribution.

**Comment by dnlyun** [comment]
> @dnlyun Do you have bandwidth to finish this PR? If not, please allow me to send a duplicate PR of yours and go forward. Thank you for your contribution.

Hi @jcwchen, you can go ahead and send the duplicate PR. Thanks
